templates/daemon.systemd.erb

```diff
@@ -2,7 +2,7 @@
 [Unit]
 Description=Prometheus <%= @name %>
 Wants=basic.target
-After=basic.target network.target
+After=basic.target network-online.target
```

Comment:
after some scrolling  through https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/ I guess we should set:

```
After=network-online.target
Wants=network-online.target
```
?

Reply:
Sorry for the late reply. Yes, I think it's better.

Reply:
I just push a updated version. Please help to check, thanks.